lib/process.js

```diff
@@ -13,6 +13,7 @@ function ProcessInfo (defaults) {
   this.root = null
   this.coverageFilename = null
   this.nodes = [] // list of children, filled by buildProcessTree()
+  this.children = [] // just uuids, not full nodes
```

Comment:
Assuming we go in the direction of not rewriting the process files I'd rather not add this here so `children` never gets added to the `processinfo/[uuid].json` files.